authui/src/authflowv2/bodyScrollLock.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller } from "@hotwired/stimulus";
+
+export class BodyScrollLockController extends Controller {
+  static targets = ["dialog"];
```

Comment:
This seems unused. And Stimulus does have something called targets https://stimulus.hotwired.dev/reference/targets